test/elevation_builder.cc

```diff
@@ -228,6 +228,7 @@ TEST(ElevationBuilder, get_tile_ids) {
   // check if config contains tile_dir
   EXPECT_TRUE(valhalla::mjolnir::get_tile_ids(config, {"/0/003/196.gph"}).empty());
 
+  // TODO: I don't understand what the following lines have to do with anything?
```

Comment:
above this uses the default config, i suppose if on your local computer you had elevation data in the default location it would pick that up and change the result of this test. to avoid that the person removes this value so that it cant happen